ui/src/views/mint/github-step/steps/github-repo-configuration/repo-configuration-body/repo-branch-commit-fields.styles.ts

```diff
@@ -0,0 +1,13 @@
+import { Text } from '@/components';
+import { styled } from '@/theme';
+
+export const TextStyles = styled(Text, {
+  maxWidth: '12.5rem',
+
+  '@media (min-width: 375px)': {
+    maxWidth: '17rem',
+  },
+  '@media (min-width: 425px)': {
+    maxWidth: '18rem',
+  },

```

Comment:
I think here it should fulfill the parent, it should not? maybe using `100%` would work better than setting fixed values.